### PR TITLE
Deployment configuration polarion IDs

### PR DESCRIPTION
**Pull request by clacroix12**
* Add polarion_ids for deployment job configurations. 
* Created separate config files for vmware vmfs and vsan

Jira: https://projects.engineering.redhat.com/browse/OCSQE-168

Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

**Comment by clacroix12** [comment]
This PR will need to be merged in sync with it's [ocs4-jenkins counterpart](https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/merge_requests/129)

File: conf/deployment/aws/upi_3az_rhel7_3m_3w.yaml [hunk 1]
```diff
@@ -13,3 +13,6 @@ ENV_DATA:
   root_disk_size: 120
   root_disk: '/dev/sda1'
   availability_zone_count: 3
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-1499'
```
Comment (dahorak) [inline conf/deployment/aws/upi_3az_rhel7_3m_3w.yaml]: I think, that there should be `OCS-1449`: *OCS-1449 - OCS Deployment on: AWS + UPI + 3 AZ + RHEL 7 + 3 W nodes*
```suggestion
    deployment_id: 'OCS-1449'
```

test Case *OCS-1499* is *OCS Upgrade - Disconnected Upgrade (W/o internet) : on VMware UPI 1 AZ + CoreOS + VMFSDatastore + 3 W nodes*.

Comment (petr-balogh) [reply conf/deployment/aws/upi_3az_rhel7_3m_3w.yaml]: good catch

Comment (clacroix12) [reply conf/deployment/aws/upi_3az_rhel7_3m_3w.yaml]: Fixed, good catch!

**Comment by petr-balogh** [comment]
@clacroix12 please fix DCO issue

**Comment by clacroix12** [comment]
> @clacroix12 please fix DCO issue

Ah, this happens if you accept a commit from github. Is there a way for us to enable signing of those commits?

**Pull request merged**